Gemfile

```
@@ -10,7 +10,7 @@ require_relative "#{base_path}lib/decidim/spam_signal/version"
 
 DECIDIM_VERSION = "0.24.3"
 
-gem "deface"
+gem "deface", '~> 1.8', '>= 1.8.1'
```

Follow your gemspec here
```suggestion
gem "deface", '>= 1.8.1'
```

---

Hi @rsdeus I see no change here ;)